modules/lib/lib-schema/src/test/resources/test/UpdateDynamicContentSchemaHandlerTest.js

```diff
@@ -21,4 +21,403 @@ exports.updateInvalidContentSchemaType = function () {
+        "type": "CONTENT_TYPE",
+        "form": [
+            {
+                "formItemType": "OptionSet",
+                "name": "radioOptionSet",
+                "label": "Single selection",
+                "expanded": false,
+                "occurrences": {
+                    "maximum": 1,
+                    "minimum": 0
+                },
+                "selection": {
+                    "maximum": 1,
+                    "minimum": 1
+                },
+                "options": [
+                    {
+                        "name": "option_1",
+                        "label": "Option 1",
+                        "default": false,
+                        "items": [
+                            {
+                                "formItemType": "Input",
+                                "name": "text-input",
+                                "label": "Name",
+                                "helpText": "Text input",
+                                "maximize": true,
+                                "inputType": "TextLine",
+                                "occurrences": {
+                                    "maximum": 1,
+                                    "minimum": 1
+                                },
+                                "default": {
+                                    "value": "something",
+                                    "type": "String"
+                                },
+                                "config": {}
+                            },
+                            {
+                                "formItemType": "ItemSet",
+                                "name": "minimum3",
+                                "label": "Minimum 3",
+                                "occurrences": {
+                                    "maximum": 0,
+                                    "minimum": 3
+                                },
+                                "items": [
+                                    {
+                                        "formItemType": "Input",
+                                        "name": "label",
+                                        "label": "Label",
+                                        "maximize": true,
+                                        "inputType": "TextLine",
+                                        "occurrences": {
+                                            "maximum": 1,
+                                            "minimum": 0
+                                        },
+                                        "config": {}
+                                    },
+                                    {
+                                        "formItemType": "Input",
+                                        "name": "value",
+                                        "label": "Value",
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9936826)
```suggestion
                                        'label': "Value",
```